client/src/components/reservation/index.tsx

```diff
@@ -0,0 +1,95 @@
+import React, { useState } from "react";
+import {
+  ReservationSection,
+  ReservationTitle,
+  ReservationInput,
+  ReservationSelect,
+  ReservationOption,
+  ReservationButton,
+  Form,
+} from "./reservation";
+import { FlexColumnContainer, FlexCenter } from "../../box/flexBox";
+import { BlockSubTitle } from "../../text/text";
+
+const Reservation = () => {
+  const [inputValue, setInputValue] = useState<string>("");
+  const [timeValue, setTimeValue] = useState<string>("2 PM");
+  const [personValue, setPersonValue] = useState<string>("2 person");
+
+  const workingHours = [
+    { id: 1, time: "10 AM" },
+    { id: 2, time: "11 AM" },
+    { id: 3, time: "12 PM" },
+    { id: 4, time: "1 PM" },
+    { id: 5, time: "2 PM" },
+    { id: 6, time: "3 PM" },
+    { id: 7, time: "4 PM" },
+    { id: 8, time: "5 PM" },
+    { id: 9, time: "6 PM" },
+    { id: 10, time: "7 PM" },
+    { id: 11, time: "8 PM" },
+    { id: 12, time: "9 PM" },
+    { id: 13, time: "10 PM" },
+    { id: 14, time: "11 PM" },
+  ];
+
+  const personCount = [
+    { id: 1, person: "2 person" },
```

Comment:
Why it's a string? It's just a number of persons. Just use an input with `type=number` and set `min, max` properties